### ISSUE TITLE
TimePicker buttons have insufficient contrast

### ISSUE THREAD

**Issue by jonahwilliams** [flutter/flutter#19915]
Currently 2.24, suggested is at least 3.00

**Comment by jonahwilliams** [comment]
Moving all button contrast issues to https://github.com/flutter/flutter/issues/19946
